content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md

```diff
@@ -266,9 +266,16 @@ This ensures that DaemonSet pods are never evicted due to these problems.
 
 ## Taint Nodes by Condition
 
-The node lifecycle controller automatically creates taints corresponding to
-Node conditions with `NoSchedule` effect.
-Similarly the scheduler does not check Node conditions; instead the scheduler checks taints. This assures that Node conditions don't affect what's scheduled onto the Node. The user can choose to ignore some of the Node's problems (represented as Node conditions) by adding appropriate Pod tolerations.
+The node controller automatically creates taints with a `NoSchedule` effect for
+[node conditions](/docs/concepts/scheduling-eviction/pod-eviction#node-conditions).
+The scheduler checks taints, not node conditions, when it makes scheduling
+decisions. This ensures that node conditions don't directly affect scheduling.
+You can ignore node conditions by adding the correct pod tolerations.
+
+For example, if the `DiskPressure` node condition is active, the control plane
+will not allow new pods to schedule on the node. If the `MemoryPressure` node
```

Comment:
Since this is properly explained above, we can say:

the control plane will add the `node.kubernetes.io/disk-pressure` taint, in order to prevent new pods to schedule on the node by default.
However, the control plane adds a toleration for the `node.kubernetes.io/memory-pressure` on Pods other than `BestEffort`.

Reply:
> However, the control plane adds a toleration for the `node.kubernetes.io/memory-pressure` on Pods other than `BestEffort`.

Is this reasonable to write:
```markdown
The control plane automatically adds a toleration for the `node.kubernetes.io/memory-pressure`
taint on Pods that have a {{< glossary_tooltip text="QoS class" term_id="qos-class" >}} other
than `BestEffort`.
Kubernetes treats Pods in the `Guaranteed` or `Burstable` QoS classes (even Pods with no
memory request set) as if they are able to cope with memory pressure.
```
?

Reply:
sgtm

Reply:
Tim, is Pod capitalized or simple? I've been lowercasing based on context but couldn't find a SG word list for this sort of stuff.